src/__tests__/generateConfluenceMarkdown.test.ts

```diff
@@ -120,3 +120,46 @@ describe('generateConfluenceMarkdown function', () => {
         expect(results).not.toContain(`|[${singleIssue[0].title}|${singleIssue[0].url}]|`);
     });
 });
+
+describe('cleanMarkdown function', () => {
+    it('should return string without altering it', () => {
+        const title1 =
+            '|[2020 upgrade edition menus too narrow|https://github.com/marmelab/react-admin/issues/7520]|8&nbsp;days|';
+        const cleanedMarkdown = cleanMarkdown(title1);
+        expect(cleanedMarkdown).toMatch(
+            '|[2020 upgrade edition menus too narrow|https://github.com/marmelab/react-admin/issues/7520]|8&nbsp;days|'
+        );
+    });
+    it('should remove a set of curly braces', () => {
+        const title = '|[{WIP} updating frontend components';
+        const url = 'https://github.com/marmelab/react-admin/issues/3450]';
+        const days = '8&nbsp;days';
+        const markdown = `${title}|${url}|${days}|`;
+        const cleanedMarkdown = cleanMarkdown(markdown);
+        expect(cleanedMarkdown).not.toContain(`${title}|${url}|${days}|`);
+        expect(cleanedMarkdown).toContain(`|[WIP updating frontend components|${url}|${days}|`);
```

Comment:
Rather than a .not.toContain and a .toContain, I think a single .toEqual should be enough.